tests/test_lazy_json_backends.py

```diff
@@ -506,3 +510,75 @@ def test_lazy_json_backends_hashmap(tmpdir):
         assert get_all_keys_for_hashmap("lazy_json") == ["blah"]
         remove_key_for_hashmap("lazy_json", "blah")
         assert get_all_keys_for_hashmap("lazy_json") == []
+
+
+class TestGithubLazyJsonBackend:
```

Comment:
Can we use functions here instead of a class for writing these tests? They are easier to maintain for me and will match the style of the current test suite.

Reply:
Resolved